### ISSUE TITLE
error when starting quake v1.08 in DOSBox-X 0.82.16

### ISSUE THREAD

**Issue by Condor-2-4-2** [joncampbell123/dosbox-x#1039]
Hello,

after starting Quake v1.08 in the latest DOSBox-X-version, I immediatly got a long error message which is shown in the screenshot bellow:

![Screen](https://user-images.githubusercontent.com/49206911/55424031-99242080-557f-11e9-82e0-738f04739a98.JPG)

I also tested the game with DOSBox 0.74-2 and here it runs without problems. 

Are there eventually wrong settings in my DOSBox-X-configfile or is there a general problem with this game? Bellow my configfile for Quake:

[dosbox-x_quake.txt](https://github.com/joncampbell123/dosbox-x/files/3035314/dosbox-x_quake.txt)

Thank you in advance!

Condor-2-4-2

**Comment by joncampbell123** [comment]
Please try Quake with core=normal

**Comment by Condor-2-4-2** [comment]
I changed core to "normal" and the game now tries to start. But after showing quake version and memory information it is crashing again with following error message:

![Screen](https://user-images.githubusercontent.com/49206911/55428140-c45f3d80-5588-11e9-807b-83264d521694.JPG)

Here the config file: 
[dosbox-x_quake.txt](https://github.com/joncampbell123/dosbox-x/files/3035490/dosbox-x_quake.txt)

**Comment by Condor-2-4-2** [comment]
... if I change one of the fault-parameters under cpu to "false", the game tries to load but stucks after initializing. I waited a few minutes but nothing happened. Otherwise there is no error message.

Here is the screenshot where it's getting stuck:

![Screen](https://user-images.githubusercontent.com/49206911/55429631-3a18d880-558c-11e9-8f72-ace5212965fe.JPG)

**Comment by joncampbell123** [comment]
The latest commit should be able to run Quake and other 386 protected mode paging DOS games in dynamic core now.

**Comment by joncampbell123** [comment]
Can you compile from source?

**Comment by Condor-2-4-2** [comment]
Hello Jon,

thank you for your replies!

Unfortunately I've not the expierience to compile binaries from source. So I tried your actual release 0.82.17 together with its contained dosbox.reference.conf today. I only changed core to "normal", the cycles to "100000", the video system to "opengl", the window resolution to "1920x1080" and completed the starting parameters for Quake under autoexec ([dosbox-x_quake.txt](https://github.com/joncampbell123/dosbox-x/files/3051880/dosbox-x_quake.txt)). I also tested the game with dynamic core parameter which you mentioned in your last comments.

But the result after starting the game with both normal or dynamic core settings was the following error message:

![Screenshot](https://user-images.githubusercontent.com/49206911/55687624-64dda500-596f-11e9-85cf-8418e28f0957.JPG)

Is there any other solution for this issue in this DOSBox-X-build?

Best regards,

Condor-2-4-2
